src/lxd_cloud/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Module for managing local LXD cloud."""
```

Comment:
Would this module be local LXD specific or LXD in general?
We might need to support remote LXD cloud in the future.

Reply:
I was thinking of having an LXD module and separating it via local/remote flag since many of the code within them will be shared. What do you think?

Reply:
The local runners needs some logic specific to it, e.g., firewall setup, running a local repo-policy-compliance, etc.

Would it be good for to have the following modules with a eventual RunnerManager interface implementation:
- openstack cloud
- lxd cloud
- local lxd

and modules for the APIs (might be a single file):
- openstack API
- lxd API
- github API

Reply:
Sounds good 👍